_src/pattern-library/sass/utilities/_variables-ltr.scss

```diff
@@ -2,6 +2,5 @@
 // edX Pattern Library: Variables (LTR-centric)
 
 // grid - configuration (susy-based)
-$susy: (
-    flow: ltr,
-);
+$grid-direction-default:    ltr !default; // default grid direction
```

Comment:
NOTE: moving the contents of this file to be plain Sass variables for easier customization and use within layout settings. This value is still getting passed to the appropriate Susy layout maps.